platforms/chibios/converters/promicro_to_kb2040/converter.mk

```
@@ -0,0 +1,7 @@
+# Adafruit KB2040 MCU settings for converting AVR projects
+MCU := RP2040
+BOOTLOADER := rp2040
+
+# These are defaults based on what has been implemented for ARM boards
+WS2812_DRIVER ?= vendor
+BACKLIGHT_DRIVER ?= software
```

```suggestion
SERIAL_DRIVER ?= vendor
WS2812_DRIVER ?= vendor
BACKLIGHT_DRIVER ?= software
```

The pio backed driver is very flexible and doesn't need any further setup.

---

Partially working through default config files like what CTPC uses, otherwise stuff was erroring out.